WorkAndFun/src/main/java/pl/infoshare/workandfun/announcements/ErrorHandlingController.java

```diff
@@ -0,0 +1,34 @@
+package pl.infoshare.workandfun.announcements;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class ErrorHandlingController implements ErrorController {
+
+
+    @RequestMapping("/error")
+    public String handleError(HttpServletRequest request) {
+
+        Object status = request.getAttribute(RequestDispatcher.ERROR_STATUS_CODE);
+
+        if (status != null) {
+            Integer statusCode = Integer.valueOf(status.toString());
```

Comment:
IntelliJ sugeruje że mozna to zmienić na prymitywny int

Reply:
poprawiłem, zaraz wrzucam kod...